docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+    </mem>
+    <pending-finalizers system="0" default="0" reference="269" classloader="0" />
+    <remembered-set count="13792" />
+  </mem-info>
+</gc-end>
+```
+
+The heap memory allocation at the end of the increment is as follows:
+
+- 97% of the *allocate* space of the *nursery* area is now available as free memory.
+- The *survivor* space of the *nursery* area is still reported as 'full' to reflect that the entire *survivor* space is reserved for GC operations during the next GC increment. 
+- The *tenure* region has 395 MB (414,331,440B) of free memory available. The *scavenge* operation copied 562 kB from the *nursery* region to the *tenure* region so less memory is now available in the *tenure* region.
+
+The *scavenge* operation successfully reclaimed memory in the *allocate* space of the nursery region by copying objects from the *allocate* space into the *survivor* space of the nursery region, and copying objects from the *survivor* space into the tenure region.
+
+The cycle ends (`<cycle-end>`). The following `<allocation-satisfied>` element indicates that the allocation request that caused the allocation failure can now complete successfully. The STW pause ends with the `<exclusive-end>` element:
+
+```xml
+<cycle-end id="12400" type="scavenge" contextid="12394" timestamp="2020-10-18T13:35:45.012" />
+<allocation-satisfied id="12401" threadId="00000000013D6900" bytesRequested="8200" />
+<af-end id="12402" timestamp="2020-10-18T13:35:45.012" threadId="00000000013D7280" success="true" from="nursery"/>
+<exclusive-end id="12403" timestamp="2020-10-18T13:35:45.012" durationms="12.319" />
+
+<!-- End of gencon partial GC cycle example -->
+```
+
+
+#### Summary of the example
+
+Analyzing the structure and elements of this example log output shows that this example global cycle has the following characteristics:
+
+- The GC cycle begins with an STW pause due to an allocation failure.
+
+- All GC operations and suboperations that are associated with this cycle occur during the STW pause
+
+- The cycle consists of only 1 GC increment, which runs a single *scavenge* operation.
+
+- The GC cycle reclaims memory in the *allocate* area of the *nursery* region by coping objects from the *allocate* area to the *survivor* area and also to the *tenure* region.
+
+### Concurrent scavenge partial GC cycle (non-default)
+
+When concurrent scavenge mode is enabled, the partial GC cycle is run as a [concurrent scavenge cycle](gc.md#concurrent-scavenge). When this mode is enabled, the scavenge partial GC cycle is divided into increments to enable the majority of the scavenge operation to run concurrently with running application threads. The concurrent increment can run while application threads run, and also while the intermediate concurrent increment of the global GC cycle runs. The interleaving of the concurrent scavenge partial GC cycle with the global cycle can be seen in the logs.
```

Comment:
I think we can avoid saying "when this mode is enabled" twice by changing the second reference to simply "This partial GC cycle...."